jsonschema/tests/test_cli.py

```diff
@@ -131,15 +185,16 @@ def test_unsuccessful_validation_multiple_instances(self):
         exit_code = cli.run(
             {
                 "validator": fake_validator(first_errors, second_errors),
-                "schema": {},
-                "instances": [1, 2],
-                "error_format": "{error.instance} - {error.message}\t",
+                "schema": "schema.json",
+                "instances": ["foo1.json", "foo2.json"],
+                "error_format": "{error.instance} - {error.message}",
+                "output": "plain",
             },
             stdout=stdout,
             stderr=stderr,
         )
         self.assertFalse(stdout.getvalue())
-        self.assertEqual(stderr.getvalue(), "1 - 9\t1 - 8\t2 - 7\t")
+        self.assertEqual(stderr.getvalue(), "1 - 9\n1 - 8\n2 - 7\n")
```

Comment:
I have to read this more carefully, but this seems like a change in behavior.

Specifically, originally it looks like there's no implicit newline in `error_format`, but now it looks like there is.

It's good I think to minimize changes to unrelated tests. Here I assume this happened because you changed the type of `arguments["instances"]` to be filenames instead of the read contents right?

Reply:
Oops, indeed that is a change, not necessarily justified.
It comes from the following lines in the `CliOutputWriter`:

```python
if not oneline_format.endswith("\n"):
    self.oneline_format += "\n"
```

I’ll just remove that so user can use the format he wants. As the original behaviour, the default format if not given by user includes this `\n`

Reply:
And I'll rename `oneline_format` to `plain_format`, more accurate.